packages/mdc-textfield/_mixins.scss

```diff
@@ -20,35 +20,398 @@
 // THE SOFTWARE.
 //
 
+@import "@material/animation/variables";
 @import "@material/density/functions";
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
 @import "@material/floating-label/mixins";
 @import "@material/line-ripple/mixins";
 @import "@material/notched-outline/mixins";
 @import "@material/notched-outline/variables";
-@import "@material/theme/mixins";
-@import "@material/shape/mixins";
+@import "@material/ripple/mixins";
+@import "@material/rtl/mixins";
 @import "@material/shape/functions";
+@import "@material/shape/mixins";
+@import "@material/theme/mixins";
+@import "@material/theme/variables";
+@import "@material/typography/mixins";
+@import "@material/typography/variables";
 @import "helper-text/mixins";
 @import "character-counter/mixins";
 @import "icon/mixins";
 @import "icon/variables";
 @import "./variables";
 @import "./functions";
 
+@mixin mdc-text-field-core-styles($query: mdc-feature-all()) {
+  @include mdc-text-field-without-ripple($query);
+  @include mdc-text-field-ripple($query);
```

Comment:
Include `mdc-text-field-ripple` before `mdc-text-field-without-ripple`. 

This order breaks textarea's ripple otherwise.

![image](https://user-images.githubusercontent.com/47312/72190790-722c3200-33ce-11ea-99ab-1d2ba186cb7a.png)